src/or/control.c

```diff
@@ -7746,6 +7753,12 @@ control_event_privcount_circuit_cell(const channel_t *chan,
                      "650 PRIVCOUNT_CIRCUIT_CELL %s\r\n",
                      event_string);
 
+  if(circ) {
```

Comment:
`keyword(` is a coding style violation, tor uses `keyword (`.
I'll fix this and similar errors in a separate commit.